### PR TITLE
breadcrumb layout / middleware updates

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/18320

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by kialj876** [comment]
Still adding tests / cleaning up

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-51-kx8hvn0o.web.app

File: btr-web/btr-common-components/cypress/e2e/pages/examples/form/email.cy.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ describe('forms -> email -> validate that email component work inside example fo
   beforeEach(() => {
     // navigate to index page and check footer and header exist
     cy.visit('/examples/form')
+    cy.wait(2000)
```
Comment (kialj876) [inline btr-web/btr-common-components/cypress/e2e/pages/examples/form/email.cy.ts:5]: for some reason this is required now with the middleware getting invoked (I think its due to cypress not having a hydration hook or something like that)

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-51-kx8hvn0o.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-51-kx8hvn0o.web.app

**Pull request merged**